config.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+options:
+  web_external_url:
```

Comment:
This config is neither needed nor desired. We are actually thinking of dropping it entirely from `prometheus-k8s`.

Reply:
@Abuelodelanada why did you close this discussion? The bogus config is still here.

Reply:
@Abuelodelanada why did you close this discussion? The bogus config is still here.

Reply:
@Abuelodelanada why did you close this discussion? The bogus config is still here.

Reply:
Forgot to delete it. Done.